pyenv.d/exec/pip-rehash.bash

```diff
@@ -6,6 +6,14 @@ if [[ $PYENV_REHASH_COMMAND =~ ^(pip|easy_install)[23](\.\d)?$ ]]; then
   PYENV_REHASH_COMMAND="${BASH_REMATCH[1]}"
 fi
 
+for (( i=1; i<$#; i++ )); do
+  next=$((i+1))
+  if [[ ${!i} == "-m" && ${!next} == "pip" ]]; then
```

Comment:
I think that just grepping `$*` (which is all arguments as one line) for `\<-m pip\>` would suffice.

Reply:
Using built-in bash regex instead. I am pretty sure that the task is so small (looking for -m pip in a shell command) that launching a grep process is not worth it performance-wise.
It's this fixed cost of setting up a new process which I think we'd rather avoid.